ui/fancy/src/js/variables.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-env browser */
+
+// This module mirrors variables.less
+
+const blockSize = [2.5, 'em'];
```

Comment:
I'm not that happy about using array for this, an object with properties `name` and `unit` would be more readable.